csharp/Ice/latency/client/Client.cs

```diff
@@ -7,7 +7,7 @@
 using ZeroC.Ice;
 
 // using statement - communicator is automatically destroyed at the end of this statement
-using var communicator = new Communicator(ref args, ConfigurationManager.AppSettings);
+await using var communicator = new Communicator(ref args, ConfigurationManager.AppSettings);
```

Comment:
After the communicator creation, we need to activate this communicator (it's a recent change):

```
// Activates the communicator. In a simple demo like this one, this activation typically does nothing. It is however
// recommended to always activate a communicator.
await communicator.ActivateAsync();
```